bundles/org.openhab.binding.modbus.kermi/src/main/resources/OH-INF/thing/xcenter-alarm-channel-types.xml

```diff
@@ -7,5 +7,9 @@
 		<item-type>Contact</item-type>
 		<label>Alarm State</label>
 		<description>Current alarm-state of your heat pump</description>
+		<tags>
+			<tag>Alarm</tag>
+			<tag>Heating</tag>
```

Comment:
I doubt it shoudl be heating. It seems to be a generic alarm state of the heatpump. It might be due to heating, but also a failing compressor, fan or no water pressure. In that case remove the Property?

Reply:
I removed the property.